### DISCUSSION TITLE
Please show year in the "Last triggered" column

### DISCUSSION THREAD

**Discussion by BoneheadFraggle** [home-assistant/frontend#21204] [Automations / scripts / scenes] [answered]
When I list which scripts & automation that are rarely used, it's unclear if it was recently - or several years ago.

Example: 
This list is sorted by Last triggered, but how can I be certain that June 12 is 2024 when Sep 10 obviously is not? 
Or is it Jun 12 2023 and Sep 10 2021?

![image](https://github.com/home-assistant/frontend/assets/2287810/2151fc6c-d216-407c-b080-2d168b3b82ca)

Suggestion: Add year. Always, or if the date is not current year.

**Comment by ildar170975** [comment]
https://github.com/home-assistant/frontend/issues/20392

**Reply by ildar170975** [reply, marked as answer]
Will be fixed by https://github.com/home-assistant/frontend/pull/23864
